### PR TITLE
Update security_bulletin_faq.rst

### PR DESCRIPTION
**Pull request by AviatrixSecurityLily**
Add FAQ for encryption and ISO 27002 certification

### CODE REVIEW

File: HowTos/security_bulletin_faq.rst [hunk 1]
```diff
@@ -216,6 +216,24 @@ How is data encrypted during transmission from source Controller to destination
...
 
+How does Aviatrix encrypt data in transit? 
+--------------------------------------------------------------------------------------------
+Aviatrix 6.5 and above implements the spiffee framework which use a PKI/X.509 strong encryption. You can read more about this framework at spiffee.io. 
+
+How does Aviatrix handle security patch?
+--------------------------------------------------------------------------------------------
+Security patch resolves software vulnerabilities and will be patched to the compatible software versions as stated in the release notes. When a patch is released, there will be a field notice to Aviatrix Controller via email. In addition, customer can send an email to securityalerts@aviatrix.com with a subject Subscribe, to subscribe to our customer mailing list. 
+
+How do I stay up to sate with the latest security vulnerabilities?
```
Comment (fpesce) [inline HowTos/security_bulletin_faq.rst:227]:
```suggestion
How do I stay up to date with the latest security vulnerabilities?
```

File: HowTos/security_bulletin_faq.rst [hunk 2]
```diff
@@ -216,6 +216,24 @@ How is data encrypted during transmission from source Controller to destination
 
 By default, data transfer is over a TCP connection with TLSv1.2 for encryption. Customers have the option to downgrade the TLS Version used due to internal dependency conflicts. You can configure this in Aviatrix Controller by clicking on Settings > Advanced > Security.
 
+How does Aviatrix encrypt data in transit? 
+--------------------------------------------------------------------------------------------
+Aviatrix 6.5 and above implements the spiffee framework which use a PKI/X.509 strong encryption. You can read more about this framework at spiffee.io. 
+
+How does Aviatrix handle security patch?
+--------------------------------------------------------------------------------------------
+Security patch resolves software vulnerabilities and will be patched to the compatible software versions as stated in the release notes. When a patch is released, there will be a field notice to Aviatrix Controller via email. In addition, customer can send an email to securityalerts@aviatrix.com with a subject Subscribe, to subscribe to our customer mailing list.
```
Comment (fpesce) [inline HowTos/security_bulletin_faq.rst:225]:
```suggestion
A security patch resolves software vulnerabilities and will be applied to the compatible software versions as stated in the release notes. When a patch is released, there will be a field notice to Aviatrix Controller via email.
```

File: HowTos/security_bulletin_faq.rst [hunk 3]
```diff
@@ -216,6 +216,24 @@ How is data encrypted during transmission from source Controller to destination
...
+--------------------------------------------------------------------------------------------
+Aviatrix 6.5 and above implements the spiffee framework which use a PKI/X.509 strong encryption. You can read more about this framework at spiffee.io. 
+
+How does Aviatrix handle security patch?
+--------------------------------------------------------------------------------------------
+Security patch resolves software vulnerabilities and will be patched to the compatible software versions as stated in the release notes. When a patch is released, there will be a field notice to Aviatrix Controller via email. In addition, customer can send an email to securityalerts@aviatrix.com with a subject Subscribe, to subscribe to our customer mailing list. 
+
+How do I stay up to sate with the latest security vulnerabilities?
+--------------------------------------------------------------------------------------------
+We recommend customer to deploy the latest image, upgrading to the latest Software version, and staying on top of any security patch released. Security vulnerabilities is complex Aviatrix's policy is to address them continously in the latest releases.
```
Comment (fpesce) [inline HowTos/security_bulletin_faq.rst:229]:
```suggestion
We recommend customers to deploy the latest image, upgrading to the latest software version, and staying on top of any security patch released. Guaranteeing security against vulnerabilities is a sustained effort and it is Aviatrix's policy to address them continuously. 
```

**Review by fpesce — CHANGES_REQUESTED** [review]

File: HowTos/security_bulletin_faq.rst (hunk 2 above)
Comment (akbhat) [reply HowTos/security_bulletin_faq.rst:225]: We should also remove the reference to SPIFFE

**Review by akbhat — APPROVED** [review]

**Review by fpesce — APPROVED** [review]

**Pull request merged**